source/macos/AdaptiveCards/AdaptiveCards/Views/ACRTextField.swift

```diff
@@ -10,11 +10,13 @@ class ACRTextField: NSTextField {
     private let config: InputFieldConfig
     private let isDarkMode: Bool?
     private let isDateTimeMode: Bool
+    private var isNumericField: Bool
     
```

Comment:
```suggestion
    private let isNumericField: Bool
    
```

Reply:
Maybe we should think about introducing `enum TextFieldMode` with `dateTime`, `numeric`, `normal` and maintain a single mode property here.